riocli/device/execute.py

```diff
@@ -29,12 +29,14 @@
 )
 @click.option('--user', default='root')
 @click.option('--shell', default='/bin/bash')
+@click.option('--deployment', default=None)
```

Comment:
question: Why do this? We don't need to modify this command.

Reply:
I thought of adding the functionality of running commands on deployment too.
but now that I think, its not needed, hence removing it